.storybook/Label.js

```diff
@@ -48,3 +48,19 @@ storiesOf('Label', module)
       <Input id="sample-input" />
     </div>
   ))
+  .add('nowrap', () => (
+    <Flex>
+      <Label.nowrap>
```

Comment:
Actually, a second thought about this, why not make the `Label` take a boolean `nowrap` prop. I think it's much cleaner.

Reply:
This sort of comes down to a static vs dynamic thing. My initial thinking was that `nowrap` and `hidden` wouldn't be a style that would change dynamically, but we don't really follow this pattern for styles, so I think you're probably right in that this would be more consistent if it were a prop